operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -29,67 +34,71 @@ This document describes the configuration of a Kubernetes NCN image. The same st
+        git clone "https://${VCS_USER}:${VCS_PASS}@api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+        ```
 
-       You will need a Git commit hash from this repo in the following step.
+        A Git commit hash from this repository is needed in the following step.
 
     1. [Create a CFS Configuration](Create_a_CFS_Configuration.md).
 
-       The first layer in the CFS configuration should be similar to this:
-
-       ```json
-       "layers": [
-       {
-         "name": "csm-ncn-workers",
-         "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
-         "playbook": "ncn-worker_nodes.yml",
-         "commit": "<git commit hash>"
-       },
-       ```
-
-       The last layer in the CFS configuration should be similar to this:
-
-       ```json
-       "layers": [
-       {
-         "name": "csm-ncn-initrd",
-         "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
-         "playbook": "ncn-initrd.yml",
-         "commit": "<git commit hash>"
-       }
-       ```
-
-1. Ensure the environment variable `$IMS_IMAGE_ID` was set during
+        The first layer in the CFS configuration should be similar to this:
+
+        ```json
+        {
+          "name": "csm-ncn-workers",
+          "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+          "playbook": "ncn-worker_nodes.yml",
+          "commit": "<git commit hash>"
+        }
+        ```
+
+        The last layer in the CFS configuration should be similar to this:
+
+        ```json
+        {
+          "name": "csm-ncn-initrd",
+          "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+          "playbook": "ncn-initrd.yml",
+          "commit": "<git commit hash>"
+        }
+        ```
+
+1. (`ncn-mw#`) Ensure that the environment variable `$IMS_IMAGE_ID` was set during
    [Import an External Image to IMS](../image_management/Import_External_Image_to_IMS.md).
 
-   ```bash
-   echo $IMS_IMAGE_ID
-   ```
+    ```bash
+    echo "${IMS_IMAGE_ID}"
+    ```
+
+1. (`ncn-mw#`) Create an image customization CFS session.
 
-1. Use the following command to create an image customization CFS session. See
-   [Create an Image Customization CFS Session](Create_an_Image_Customization_CFS_Session.md) for additional information
-   on creating an image customization CFS session.
+    See [Create an Image Customization CFS Session](Create_an_Image_Customization_CFS_Session.md) for additional information
+    on creating an image customization CFS session.
 
-   ```bash
-   cray cfs sessions create --name ncn-image-customization-session \
-       --configuration-name ncn-image-customization \
-       --target-definition image --format json \
-       --target-group Management_Worker $IMS_IMAGE_ID
-   ```
+    ```bash
```

Comment:
Add step to delete ncn-image-customization-session, if present, before creating a new session

Reply:
Instead I changed it so that it includes a timestamp in the name